sct-commons/src/test/java/org/lfenergy/compas/sct/commons/scl/ied/AccessPointAdapterTest.java

```diff
@@ -222,15 +222,15 @@ void checkLimitationForBindedIEDControls_should_fail_three_error_messages() thro
         //Then
         assertThat(sclReportItems).hasSize(3)
                 .extracting(SclReportItem::getMessage)
-                .containsExactlyInAnyOrder("The Client IED IED_NAME1 subscribes to too much GOOSE Control Blocks.",
-                        "The Client IED IED_NAME1 subscribes to too much SMV Control Blocks.",
-                        "The Client IED IED_NAME1 subscribes to too much REPORT Control Blocks.");
+                .containsExactlyInAnyOrder("The Client IED IED_NAME1 subscribes to too much GOOSE Control Blocks. 3 > (max) 2",
```

Comment:
Update issue or roolback changes

Reply:
I updated the issue in Jira and Github for all error messages.